recipe/meta.yaml

```diff
@@ -8,34 +8,48 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 118eb54fd2794049b810893653b20952349df6d3bc1764e7facd8a18064fa9b0
+  sha256: 52f682ba4efe29dccb38ff0fe5bac8a23007d0780ff92a8b85af64bc4fc74d72
 
 build:
-  noarch: python
+  skip: true  # [py<37]
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
-    - python >=3.7
-    - setuptools_scm
+    - python
+    - wheel
+    - hatchling
+    - hatch-vcs
   run:
-    - python >=3.7
+    - python
     - importlib-metadata
     - wcwidth
   run_constrained:
+    # Following from conda-forge
```

Comment:
Conda-forge has this constraint to prevent `ptable` from being installed with `prettytable`. `ptable` is a fork of `prettytable` that uses the same name. `ptable` isn't on defaults, so we might consider removing this.

See relevant conda-forge PR here: https://github.com/conda-forge/prettytable-feedstock/pull/13

Reply:
I'm not sure that we will add `ptable` to the defaults channel but in the future other packages can depend on it. This run_constrained will prevent any conflicts.